data/styles/default/include/priorities_4_overlays.prio.txt

```diff
@@ -599,7 +599,9 @@ landuse-allotments                                  # caption z15- (also has are
 landuse-cemetery                                    # icon z15- (also has caption(optional) z15-, area z14-)
 landuse-cemetery-christian                          # icon z15- (also has caption(optional) z15-, area z14-)
 man_made-water_tap                                  # icon z16- (also has caption(optional) z18-)
+man_made-water_tap-drinking_water_no                # icon z18- (also has caption(optional) z18-)
 man_made-water_well                                 # icon z16- (also has caption(optional) z18-)
+man_made-water_well-drinking_water_no               # icon z18- (also has caption(optional) z18-)
```

Comment:
please make non-drinkable sources less prioritized, e.g. put to 1200

Reply:
Moved to 1200